modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/dto/v1_0/converter/StructuredContentDTOConverter.java

```diff
@@ -232,6 +232,23 @@ private Map<Locale, String> _filterDescriptionMap(
 		);
 	}
 
+	private boolean _notContainRenderedContentValue(UriInfo uriInfo) {
+		MultivaluedMap<String, String> parameters =
+			uriInfo.getQueryParameters();
+
+		if ((parameters == null) || parameters.isEmpty()) {
```

Comment:
@slnn Is it possible for this parameter map to be null?

Reply:
Hi @dantewang

The parameter map is possible to be null, please see the implementation for getQueryParameters() in HeadlessDiscoveryOpenAPIResourceImpl which is an implementation of UriInfo

```
@Override
			public MultivaluedMap<String, String> getQueryParameters() {
				return null;
			}
```